engine/src/main/java/org/terasology/rendering/world/WorldRendererLwjgl.java

```diff
@@ -78,57 +76,37 @@
 import static org.lwjgl.opengl.GL11.glPushMatrix;
 
 /**
- * The world of Terasology. At its most basic the world contains chunks (consisting of a fixed amount of blocks)
- * and the player.
- * <p/>
- * The world is randomly generated by using a bunch of Perlin noise generators initialized
- * with a favored seed value.
- *
  * @author Benjamin Glatzel <benjamin.glatzel@me.com>
  */
 public final class WorldRendererLwjgl implements WorldRenderer {
-    public static final int VERTICAL_SEGMENTS = CoreRegistry.get(Config.class).getSystem().getVerticalChunkMeshSegments();
 
     private static final int SHADOW_FRUSTUM_BOUNDS = 500;
 
-    /* WORLD PROVIDER */
+    private final int verticalMeshSegments = CoreRegistry.get(Config.class).getSystem().getVerticalChunkMeshSegments();
+
+    private final Skysphere skysphere;
     private final WorldProvider worldProvider;
+    private final RenderableWorld renderableWorld;
 
-    /* PLAYER */
     private LocalPlayer player;
 
-    /* CAMERA */
-    private Camera activeCamera;
+    private final Camera activeCamera;
```

Comment:
This seems a little odd - activeCamera implies the camera can change, but final will prevent this?